beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4798,6 +4832,29 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         Ok((block, state))
     }
 
+    fn compute_blob_kzg_proofs(
+        kzg: &Arc<Kzg>,
+        blobs: &Blobs<<T as BeaconChainTypes>::EthSpec>,
```

Comment:
This is using `Blobs` which seems to be the same as `BlobSidecars`. I _think_ since they are just type aliases that it should be ok to use the other one (or just one in any case)

Reply:
Oh yea I'll update this to `BlobSidecarList`

Reply:
Oh wait they're different `Blobs` = `VariableList<Blob>`  and `BlobSidecars` (now named `BlobSidecarList`) = `VariableList<BlobSidecar>`